pom.xml

```diff
@@ -238,6 +234,22 @@
       <artifactId>bson</artifactId>
       <version>5.0.0</version>
     </dependency>
+    <!-- clunven fixing cve https://datastax.jira.com/browse/MC-1977 -->
+    <dependency>
+      <groupId>io.smallrye</groupId>
+      <artifactId>smallrye-fault-tolerance-core</artifactId>
+      <version>6.9.0</version>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
```

Comment:
Are these 2 (smallrye-fault-tolerance-core, vertx-core) overrides for versions Quarkus uses? Maybe add a comment saying that, so we can remove overrides when Quarkus updates.

Reply:
Upgrading to quarkus `3.24.4` and not both versions are already pulled properly no need to add a version